mslib/msui/constants.py

```diff
@@ -48,6 +48,14 @@
 
 MSS_SETTINGS = os.getenv('MSS_SETTINGS', os.path.join(MSS_CONFIG_PATH, "mss_settings.json"))
 
+if '://' in MSS_SETTINGS:
+    if not _fs.exists('mss_settings.json'):
+        _fs.writetext("mss_settings.json", "{}")
```

Comment:
Shouldn't people with an fs path also be able to set another filename than mss_settings.json?

Reply:
yes ;)